classes/notification/PKPNotification.inc.php

```diff
@@ -62,15 +62,13 @@
 define('NOTIFICATION_TYPE_PENDING_INTERNAL_REVISIONS',		0x1000015);
 define('NOTIFICATION_TYPE_PENDING_EXTERNAL_REVISIONS',		0x1000016);
 define('NOTIFICATION_TYPE_COPYEDIT_ASSIGNMENT',			0x1000017);
-define('NOTIFICATION_TYPE_ALL_REVIEWS_IN',			0x1000018);
```

Comment:
Looks good -- but you may as well remove the `tableExists` condition, unless it's useful for some specific purpose. That table exists for all supported versions of OJS we're upgrading.